examples/cop-dem-glo-30/Copernicus_DSM_COG_10_N53_00_W115_00_DEM/Copernicus_DSM_COG_10_N53_00_W115_00_DEM.json

```diff
@@ -113,6 +124,129 @@
       "roles": [
         "data"
       ]
+    },
+    "src": {
+      "href": "../../tests/data-files/external/PREVIEW/Copernicus_DSM_COG_10_N53_00_W115_00_SRC.kml",
+      "type": "application/vnd.google-earth.kml+xml",
+      "title": "Source mask",
+      "description": "Source Mask",
+      "role": [
+        "data",
+        "data-mask"
+      ]
+    },
+    "dem_ql": {
+      "href": "../../tests/data-files/external/PREVIEW/Copernicus_DSM_COG_10_N53_00_W115_00_DEM_QL.tif",
+      "type": "image/tiff",
```

Comment:
are these any more specific than just tifs (geotiff or cog)?

Reply:
Looks like they are GeoTiff but not COG, I'll update.